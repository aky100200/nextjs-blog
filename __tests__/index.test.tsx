import renderer from 'react-test-renderer'
import React from 'react'

import App from '../pages/index'

test('AppTitle', () => {
  const component = renderer.create(
    <App
      allPostsData={[
        { date: '2020-01-01', title: 'test_title', id: 'test_id' },
      ]}
    />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
