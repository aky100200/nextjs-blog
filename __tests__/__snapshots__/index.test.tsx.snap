// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle 1`] = `
<div>
  <header>
    <img
      alt="aky100200"
      className="undefined undefined"
      src="/images/profile.jpg"
    />
    <h1>
      aky100200
    </h1>
  </header>
  <main>
    <section>
      <h2>
        Profile
      </h2>
      <p>
        技術スキルが足りないまま6年目になってしまったSIerのSE。
        <br />
        言語:Java/JS/PHP/TS
        <br />
        FW:SpringBoot/AngularJS/React
        <br />
        クラウド:Azure/AWS
        <br />
        保有資格：基本情報/応用情報
      </p>
    </section>
    <section
      className="undefined undefined"
    >
      <h2>
        Blog
      </h2>
      <ul>
        <li>
          <a
            href="/posts/test_id"
            id="test_id"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            test_title
          </a>
          <br />
          <small>
            <time
              dateTime="2020-01-01"
            >
              January 1, 2020
            </time>
          </small>
        </li>
      </ul>
    </section>
    <section
      className="undefined undefined"
    >
      <h2>
        Questionnaire
      </h2>
      <a
        href="https://forms.gle/4r3YHkhRjDcAnyGT6"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        アンケートページ
      </a>
    </section>
  </main>
</div>
`;
